test/browser/spec/ol/render/webgl/MixedGeometryBatch.test.js

```diff
@@ -998,30 +998,30 @@ describe('MixedGeometryBatch', function () {
             [0, 1],
             [2, 3],
             [4, 5],
-            [6, 7],
+            [-6, 7],
           ],
           [
             [20, 21],
             [22, 23],
-            [24, 25],
+            [24, -25],
           ],
         ],
         [
           [
             [8, 9],
             [10, 11],
-            [12, 13],
+            [-12, 13],
           ],
           [
             [30, 31],
             [32, 33],
-            [34, 35],
+            [34, -35],
           ],
           [
             [40, 41],
             [42, 43],
             [44, 45],
-            [46, 47],
+            [46, -47],
```

Comment:
These coordinates have been changed so that the winding orders are correct for the outer and inner rings (before that not much attention had been given to it)